mcux/mcux-sdk/drivers/usdhc/fsl_usdhc.c

```diff
@@ -1222,11 +1222,11 @@ status_t USDHC_SetADMA1Descriptor(
 
     uint32_t miniEntries, startEntries = 0UL,
                           maxEntries = (admaTableWords * sizeof(uint32_t)) / sizeof(usdhc_adma1_descriptor_t);
-    usdhc_adma1_descriptor_t *adma1EntryAddress = (usdhc_adma1_descriptor_t *)(uint32_t)(admaTable);
+    usdhc_adma1_descriptor_t *adma1EntryAddress = (usdhc_adma1_descriptor_t *)(uintptr_t)(admaTable);
```

Comment:
does this change need to be directed now to mcux-sdk-ng folder?

@ZhaoxiangJin,